mindsdb/libs/controllers/transaction.py

```diff
@@ -171,7 +171,7 @@ def _execute_learn(self):
             self._call_phase_module(clean_exit=True, module_name='DataSplitter')
 
             self._call_phase_module(clean_exit=True, module_name='DataTransformer', input_data=self.input_data)
-
+            exit()
```

Comment:
?

Reply:
@George3d6 ^

Reply:
Debugging code, not supposed to me merged.